pkg/controller/usersshkeysagent/usersshkeys_controller.go

```diff
@@ -47,7 +47,7 @@ import (
 )
 
 const (
-	operatorName = "usersshkeys-controller"
+	operatorName = "kkp-usersshkeys-controller"
```

Comment:
same as the comment for machinedeployment controller

Reply:
Again, it's just the name of the controller which is not following our conventions.